### DISCUSSION TITLE
能否增加获取群消息历史的条数

### DISCUSSION THREAD

**Discussion by zht163** [Mrs4s/go-cqhttp#838] [Ideas]
目前获取群消息历史的条数最多是20条，如果想要批量撤回某个人的最近的多少条发言，这20条就太少了,需要通过上一次获取的历史记录里最开始的message_seq来重复获取历史记录。

看能否增加获取群消息历史的条数，或者接受请求时指定的需要返回的消息数量(这个数量有最大值，比如100/150)

**Comment by Mrs4s** [comment]
这个20条是TX的限制

**Reply by zht163** [reply]
啊这，打扰了
